bentoml/start.py

```diff
@@ -149,18 +153,19 @@ def start_runner_server(
         sockets=list(circus_socket_map.values()),
     )
 
-    try:
-        arbiter.start(
-            cb=lambda _: logger.info(  # type: ignore
-                'Starting RunnerServer from "%s"\n running on http://%s:%s (Press CTRL+C to quit)',
-                bento_identifier,
-                host,
-                port,
-            ),
-        )
-    finally:
-        if uds_path is not None:
-            shutil.rmtree(uds_path)
+    with track_serve(svc, production=True, component=RUNNER):
+        try:
+            arbiter.start(
+                cb=lambda _: logger.info(  # type: ignore
+                    'Starting RunnerServer from "%s"\n running on http://%s:%s (Press CTRL+C to quit)',
+                    bento_identifier,
+                    host,
+                    port,
+                ),
+            )
+        finally:
+            if uds_path is not None:
+                shutil.rmtree(uds_path)
```

Comment:
I don't think uds_path is not used here.

Reply:
I did not intend to change this part of the logic. @bojiang do you think the the `uds_path` logic can be removed?